README.md

```diff
@@ -52,6 +52,7 @@ like `KEYS *` (see
   to send a EOF to exit IRedis.
 - <kbd>Ctrl</kbd> + <kbd>R</kbd> to open **reverse-i-search** to search through
   your command history.
+- <kbd>Ctrl</kbd> + <kbd>Z</kbd> to suspend IRedis.
```

Comment:
This is not a "feature", more like a common function for command line tools, so I prefer put this into CHANGELOG.md instead of readme.